isamples_metadata/GEOMETransformer.py

```diff
@@ -431,6 +431,10 @@ def related_resources(self) -> typing.List[typing.Dict]:
     def child_transformers(self) -> typing.List[GEOMEChildTransformer]:
         return self._child_transformers
 
+    def last_updated_time(self) -> typing.Optional[typing.AnyStr]:
+        # This isn't available in GEOME
+        return None
+
```

Comment:
Another unrelated refactoring breakage discovered because I didn't run the unit tests until now.